script.module.kodiswift/lib/kodiswift/constants.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""
+kodiswift.constants
+--------------------
+
+This module contains some helpful constants which ease interaction
+with Kodi.
+
+:copyright: (c) 2012 by Jonathan Beluch
+:license: GPLv3, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+__all__ = ['SortMethod']
+
+
+class SortMethod(object):
+    """Static class to hold all of the available sort methods. The prefix
+    of 'SORT_METHOD_' is stripped.
+
+    e.g. SORT_METHOD_TITLE becomes SortMethod.TITLE
+    """
+    ALBUM = 13
+    ALBUM_IGNORE_THE = 14
+    ARTIST = 11
+    ARTIST_IGNORE_THE = 12
+    BITRATE = 40
+    CHANNEL = 38
+    COUNTRY = 16
+    DATE = 3
+    DATEADDED = 19
+    DATE_TAKEN = 41
+    DRIVE_TYPE = 6
+    DURATION = 8
+    EPISODE = 22
+    FILE = 5
+    FULLPATH = 32
+    GENRE = 15
+    LABEL = 1
+    LABEL_IGNORE_FOLDERS = 33
+    LABEL_IGNORE_THE = 2
+    LASTPLAYED = 34
+    LISTENERS = 36
+    MPAA_RATING = 28
+    NONE = 0
+    PLAYCOUNT = 35
+    PLAYLIST_ORDER = 21
+    PRODUCTIONCODE = 26
+    PROGRAM_COUNT = 20
+    SIZE = 4
+    SONG_RATING = 27
+    STUDIO = 30
+    STUDIO_IGNORE_THE = 31
+    TITLE = 9
+    TITLE_IGNORE_THE = 10
+    TRACKNUM = 7
+    UNSORTED = 37
+    VIDEO_RATING = 18
+    VIDEO_RUNTIME = 29
+    VIDEO_SORT_TITLE = 24
+    VIDEO_SORT_TITLE_IGNORE_THE = 25
+    VIDEO_TITLE = 23
+    VIDEO_YEAR = 17
```

Comment:
What strings are you talkin about? There are no strings related to this anywhere. Nowhere.
I rather have some more module add-ons than having to review twice as much code (or even more). I even urge addon devs to create frameworks when they do lot of different add-ons. That also helps to properly separate your code, so you dont run into danger to have circular imports for example when developing more complex add-ons. If you hate something, give a proper reason why. Havin some module add-ons do not hurt at all.
You really put up trademark concerns? Then I might also drop my own framework (kodi65) as well, and some other add-on devs would have to do that as well.
Your second point might be valid. @Sinap, do you have an add-on in the pipeline which makes use of this?
If this is the way we review add-ons, then I wil step away from it. Someone puts up some great work and then has to listen to this....